packages/next-common/components/fellowship/core/members/actions/more/submitEvidenceItem/popup.jsx

```diff
@@ -0,0 +1,151 @@
+        "cursor-pointer",
+      )}
+      onClick={onClick}
+    >
+      <div className="flex flex-col grow">
+        <span className="text14Medium text-textPrimary">{title}</span>
+        <span className="text12Medium text-textTertiary">{description}</span>
+      </div>
+      <div>
+        {checked ? (
+          <SystemRadioButtonOn className="[&_path]:fill-theme500" />
+        ) : (
+          <SystemRadioButtonOff />
+        )}
+      </div>
+    </div>
+  );
+}
+
+function Content() {
+  const { onClose } = usePopupParams();
+  const dispatch = useDispatch();
+  const address = useRealAddress();
+  const { component } = useSigner("Address");
+  const [evidence, setEvidence] = useState("");
+  const [wish, setWish] = useState("retention");
+  const { uploading, upload } = useUploadToIpfs();
+
+  const api = useContextApi();
+
+  const getTxFunc = useCallback(async () => {
+    const { error, result } = await upload(
+      new File([evidence], `evidence-${address}-${wish}.txt`, {
+        type: "text/plain",
+      }),
+    );
+    if (error) {
+      dispatch(newErrorToast("Failed to upload evidence to IPFS"));
+      return;
+    }
+    const { cid } = result;
+
+    let digest;
+    try {
+      digest = CID.parse(cid).toV0().multihash.digest;
+    } catch (e) {
+      dispatch(newErrorToast("Failed to parse CID digest"));
+      return;
+    }
+
+    const hexDigest = "0x" + Buffer.from(digest).toString("hex");
+    return api.tx.fellowshipCore?.submitEvidence(wish, hexDigest);
+  }, [api, address, upload, evidence, wish, dispatch]);
+
+  const fnWaitSync = useWaitSyncBlock("Evidence submitted", () => {
+    dispatch(incFellowshipCoreMembersTrigger());
+  });
+  const onSubmit = (_, blockHash) => blockHash && fnWaitSync(blockHash);
+
+  return (
+    <>
+      {component}
+      <div>
+        <PopupLabel
+          text={
+            <span>
+              Wish <span className="text-red500">*</span>
+            </span>
+          }
+        />
+        <div className="flex gap-[8px]">
+          <WishChoice
+            title="Retention"
+            description="Retain the current rank"
+            checked={wish === "retention"}
+            onClick={() => setWish("retention")}
+          />
+          <WishChoice
+            title="Promotion"
+            description="Promote to a higher rank"
+            checked={wish === "promotion"}
+            onClick={() => setWish("promotion")}
+          />
+        </div>
+      </div>
+      <div>
+        <PopupLabel
+          text={
+            <span>
+              Evidence <span className="text-red500">*</span>
+            </span>
+          }
+        />
+        <Editor
+          value={evidence}
+          onChange={setEvidence}
+          contentType={"markdown"}
+          setContentType={() => {}}
+          loadSuggestions={() => []}
+          minHeight={100}
+          previewerPlugins={[]}
+          setQuillRef={() => {}}
```

Comment:
Can we ignore attributes with empty functions? Are there default values in `Editor` component?